src/PrestaShopBundle/PrestaShopBundle.php

```diff
@@ -72,6 +85,8 @@ public function build(ContainerBuilder $container)
     {
         $container->addCompilerPass(new DynamicRolePass());
         $container->addCompilerPass(new PopulateTranslationProvidersPass());
+        $container->addCompilerPass(new LoadServicesFromModulesPass(), PassConfig::TYPE_BEFORE_OPTIMIZATION, self::LOAD_MODULE_SERVICES_PASS_PRIORITY);
+        $container->addCompilerPass(new LoadServicesFromModulesPass($this->appType), PassConfig::TYPE_BEFORE_OPTIMIZATION, self::LOAD_MODULE_SERVICES_PASS_PRIORITY);
```

Comment:
This is super dangerous. The DI should never know if we are in a frontend or a backend context. This information should come from the request for example, but should not be used inside a compiler pass.